can/interfaces/kvaser/canlib.py

```diff
@@ -573,35 +573,39 @@ def __init__(
         )
         canSetBusOutputControl(self._write_handle, can_driver_mode)
 
+        self._is_filtered = False
+        super().__init__(
+            channel=channel,
+            can_filters=can_filters,
+            **kwargs,
+        )
+
+        # activate channel after CAN filters were applied
+        log.debug("Go on bus")
+        if not self.single_handle:
+            canBusOn(self._read_handle)
+        canBusOn(self._write_handle)
+
+        # timestamp must be set after bus is online
```

Comment:
To be picky, it would be nice with a comment on why here: 
_...since otherwise kvReadTimer seem to return erroneous values_